website/docs/r/oss_bucket.html.markdown

```diff
@@ -139,6 +151,7 @@ The following arguments are supported:
 * `lifecycle_rule` - (Optional) A configuration of [object lifecycle management](https://www.alibabacloud.com/help/doc-detail/31904.htm) (documented below).
 * `policy` - (Optional, Available in 1.41.0) Json format text of bucket policy [bucket policy management](https://www.alibabacloud.com/help/doc-detail/100680.htm) (documented below).
 * `storage_class` - (Optional, ForceNew) The [storage class](https://www.alibabacloud.com/help/doc-detail/51374.htm) to apply. Can be "Standard", "IA" and "Archive". Defaults to "Standard".
+* `server_side_encryption_rule` - (Optional) A configuration of server-side encryption (documented below).
```

Comment:
Please add a tag for the `server_side_encryption_rule - (Optional, Available in 1.45.0+)`

Reply:
done